project-generated-code-backend/Backend/Model/Hospital/Equipment.cs

```diff
@@ -34,15 +34,7 @@ public Equipment(String serialNumber, string name, string id) : base()
             this.id = id;
         }
 
-        [JsonConstructor]
-        public Equipment(String serialNumber, string name, string id, string roomId) : base()
-        {
-            this.SerialNumber = serialNumber;
-            this.name = name;
-            this.id = id;
-            this.roomId = roomId;
-        }
-
+       
```

Comment:
Deleting just the annotation was fine, idk if this Constructor is used by any other project